ui/src/components/CardListItem.vue

```diff
@@ -1,21 +1,168 @@
 
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by location',
+                        data: value
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAexs: [{
+                            ticks: {
+                                beginAtZero: true
+                            }
+                        }]
+                    }
+                }
+            });
+        }
+        else if (this.employmentTypes) {
+               for (let i = 0; i < this.employmentTypes.length; i++) {
+                label.push(this.employmentTypes[i].key);
+                value.push(this.employmentTypes[i].doc_count);
+            }
+
+            const ctx = document.getElementById('employmentType-chart');
+
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by Employment Types',
+                        data: value
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAexs: [{
+                            ticks: {
+                                beginAtZero: true
+                            }
+                        }]
+                    }
+                }
+            });
+        }
+        else if (this.skills) {
+               for (let i = 0; i < this.skills.length; i++) {
+                label.push(this.skills[i].key);
+                value.push(this.skills[i].doc_count);
+            }
+
+            const ctx = document.getElementById('skill-chart');
+
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by Skills',
+                        data: value
+                    }]
+                },
+                options: {
+                    scales: {
+                        yAexs: [{
+                            ticks: {
+                                beginAtZero: true
+                            }
+                        }]
+                    }
+                }
+            });
+        }
+        else if (this.isRemote) {
+               for (let i = 0; i < this.isRemote.length; i++) {
+                label.push(this.isRemote[i].key);
+                value.push(this.isRemote[i].doc_count);
+            }
+
+            const ctx = document.getElementById('remote-chart');
+
+            new Chart(ctx, {
+                type: 'horizontalBar',
+                data: {
+                    labels: label,
+                    datasets: [{
+                        label: 'Job numbers by Remote Eligible',
```

Comment:
It's not Remote Eligible.. it's just Remote. Some companies only offer remote and don't have option for on-site.